src/scales/scale.linearbase.js

```diff
@@ -204,7 +232,8 @@ export default class LinearScaleBase extends Scale {
       min: opts.min,
       max: opts.max,
       precision: tickOpts.precision,
-      stepSize: tickOpts.stepSize
+      step: tickOpts.stepSize,
```

Comment:
I was tempted to rename `tickOpts.stepSize` to `tickOpts.step`

Reply:
I'd agree with that. Probably better done separately though.